### PR TITLE
Fixes to forms

### PR DESCRIPTION
**Pull request by lili2311**
[Issue 11](https://github.com/womenhackfornonprofits/pimpmycause-rebuild/issues/11)

### CODE REVIEW

**Comment by tanyapowell** [comment]
is this for @rimpey?

**Comment by lili2311** [comment]
anyone :)

File: frontend/src/styles/layout/_forms.scss [hunk 1]
```diff
@@ -35,6 +38,6 @@
 #newsletter {
 	.button {
 		background-color: $colr-btn-newsleter;
-		width: 10em;
+		//width: 10em;
```
Comment (tanyapowell) [inline frontend/src/styles/layout/_forms.scss]: Probably best to delete commented code

**Comment by lili2311** [comment]
deleted.

**Pull request merged**